rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -822,7 +822,20 @@ public enum BridgeMethods {
         activations -> activations.isActive(RSKIP271),
         fixedPermission(false),
         CallTypeHelper.ALLOW_STATIC_CALL
-    );
+    ),
+    // Union Bridge Methods
+    SET_UNION_BRIDGE_CONTRACT_ADDRESS_FOR_TESTNET(
+        CallTransaction.Function.fromSignature(
+            "setUnionBridgeContractAddressForTestnet",
+            new String[]{"address"},
+            new String[]{"int"}
+        ),
+        fixedCost(24000L),
```

Comment:
Is the final value?

Reply:
`TODO` added to define the final value later.